python/lsst/afw/math/_spatialCell.py

```diff
@@ -37,6 +38,25 @@ def spatialCellCandidateIter(self):
 SpatialCellCandidateIterator.__iter__ = spatialCellCandidateIter
 
 
+def spatialCellCandidateRepr(self):
+    return (f"center=({self.getXCenter()},{self.getYCenter()}), "
+            f"status={self.getStatus().name}, rating={self.getCandidateRating()}")
+
+
+SpatialCellCandidate.__repr__ = spatialCellCandidateRepr
```

Comment:
Might want to [`lsst.utils.continueClass`](https://pipelines.lsst.io/v/weekly/py-api/lsst.utils.continueClass.html#lsst.utils.continueClass) for the content of this module.

Reply:
I was just continuing the pattern that had been set up, but you're right. I'll rework it.